compactor/periodic.go

```diff
@@ -113,8 +113,8 @@ func (t *Periodic) Resume() {
 	t.paused = false
 }
 
-func (t *Periodic) getRev(h int) (int64, []int64) {
-	i := len(t.revs) - int(time.Duration(h)*time.Hour/checkCompactionInterval)
+func (t *Periodic) getRev() (int64, []int64) {
```

Comment:
pass in n? probably we do not need the global defined const `n`.

Reply:
the `n` can be used in testing. that's why I had a const `n`.

Reply:
ah. ok. makes sense.